src/Orchard.Web/Modules/Orchard.Users/Controllers/AccountController.cs

```diff
@@ -204,7 +204,7 @@ public class AccountController : Controller {
 
             _userService.SendLostPasswordEmail(username, nonce => Url.MakeAbsolute(Url.Action("LostPassword", "Account", new { Area = "Orchard.Users", nonce = nonce }), siteUrl));
 
-            _orchardServices.Notifier.Information(T("Check your e-mail for the confirmation link."));
+            _orchardServices.Notifier.Information(T("If you are an existing user, we will immediately send you an email with a link to reset your password."));
```

Comment:
this doesn't really add any value as the email the user submitted isn't provided back to them, so they won't know whether they have made a typo. Something better would be:
```
T("Check your e-mail for the confirmation link. We will be sending the message to: {0}", email)